config-model/src/test/java/com/yahoo/vespa/model/content/StorageClusterTest.java

```diff
@@ -189,7 +189,7 @@ void merge_throttler_memory_limit_config_has_expected_defaults() {
         var config = configFromProperties(new TestProperties());
         var limit = config.merge_throttling_memory_limit();
 
-        assertEquals(-1L, limit.max_usage_bytes()); // TODO change default
+        assertEquals(0L, limit.max_usage_bytes()); // TODO change default
```

Comment:
The `TODO` can probably be removed with this change

Reply:
Fixed